backends/gdx-backends-gwt/src/com/badlogic/gwtref/gen/ReflectionCacheSourceCreator.java

```diff
@@ -517,16 +517,16 @@ private String createTypeGenerator (JType t) {
 		}
 		
 		if (c != null && c.getImplementedInterfaces() != null) {
-			interfaces = "new HashSet<Class>(Arrays.asList(";
+			interfaces = new StringBuilder("new HashSet<Class>(Arrays.asList(");
 			boolean used = false;
 			for (JType i : c.getImplementedInterfaces()) {
 				if (!isVisible(i) || i.equals(t)) continue;
-				if (used) interfaces += ", ";
-				interfaces += i.getErasedType().getQualifiedSourceName() + ".class";
+				if (used) interfaces.append(", ");
+				interfaces.append(i.getErasedType().getQualifiedSourceName()).append(".class");
 				used = true;
 			}
 			if (used)
-				interfaces += "))";
+				interfaces.append("))");
```

Comment:
I'm not familiar with the GWT reflection pipeline, but this seems to generate code, so I assume it runs at compile time.

In either case, this whole class is so full of string concatenation, that this change is pointless and just makes this inconsistent.